haystack/nodes/prompt/prompt_node.py

```diff
@@ -1012,37 +1069,59 @@ def _flatten_inputs(
             input_queries = [None]
             input_invocation_contexts = [None]
 
+        if prompt_templates is not None:
+            if len(prompt_templates) != len(input_queries):
+                raise ValueError("The input variables prompt_templates and queries should have the same length.")
+            input_prompt_templates = prompt_templates
+        else:
+            input_prompt_templates = [None] * len(input_queries)
```

Comment:
What's this going to do? That's just some default value we'll handle ok @tstadel ?

Reply:
Yes exactly